frontend/packages/helm-plugin/src/components/details-page/history/HelmReleaseHistory.tsx

```diff
@@ -28,16 +28,16 @@ const HelmReleaseHistory: React.FC<HelmReleaseHistoryProps> = ({
   const helmReleaseName = params.name;
   const [revisionsLoaded, setRevisionsLoaded] = React.useState<boolean>(false);
   const [loadError, setLoadError] = React.useState<string>();
-  const [revisions, setRevisions] = React.useState([]);
+  const [revisions, setRevisions] = React.useState<HelmRelease[]>([]);
   const memoizedObj = useDeepCompareMemoize(obj);
   const { t } = useTranslation();
 
   React.useEffect(() => {
     let destroyed = false;
-    fetchHelmReleaseHistory(helmReleaseName, namespace)
+    fetchHelmReleaseHistory(helmReleaseName || '', namespace || '')
```

Comment:
Same question here. It feels like this is a problem if either of these values is false-y.